include/openssl/kdf.h

```diff
@@ -21,6 +21,16 @@
 extern "C" {
 #endif
 
+// CRYPTO_tls1_prf calculates |out_len| bytes of the TLS PRF, using |digest|,
+// and writes them to |out|. It returns one on success and zero on error.
+// TLS 1.2: https://datatracker.ietf.org/doc/html/rfc5246#section-5
+// TLS 1.{0,1}: https://datatracker.ietf.org/doc/html/rfc4346#section-5
+OPENSSL_EXPORT int CRYPTO_tls1_prf(const EVP_MD *digest,
```

Comment:
Does this work for 1.3? Should the name/documentation be updated to make that clear? Also do we have a 1.3 specific function?